.github/compatibility-test-matrices/unreleased/ica-queries.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Adding also the ICA queries e2e test for unreleased, release/v7.5.x (pending investigation by SDK team to see if we can finally back port this feature) and release/v8.3.x.